var/spack/repos/builtin/packages/r-jsonlite/package.py

```diff
@@ -38,10 +38,10 @@ class RJsonlite(Package):
     use with dynamic data in systems and applications."""
 
     homepage = "https://github.com/jeroenooms/jsonlite"
-    url      = "https://cran.r-project.org/src/contrib/jsonlite_0.9.21.tar.gz"
+    url      = "https://cran.r-project.org/src/contrib/jsonlite_1.0.tar.gz"
     list_url = "https://cran.r-project.org/src/contrib/Archive/jsonlite"
 
-    version('0.9.21', '4fc382747f88a79ff0718a0d06bed45d')
+    version('1.0', 'c8524e086de22ab39b8ac8000220cc87')
 
```

Comment:
You do not have to change the url, just adding the new version and md5 is sufficient. Also, I think there is a general preference to put the newer version on top.